components/organisms/PokemonList.tsx

```diff
@@ -0,0 +1,96 @@
+import { PokemonSummary } from '@/types/pokemon';
+import SearchableContent from './SearchableContent';
+import Card, { CardProps } from '../molecules/Card';
+import { IMAGE_BASE_URL, WIKI_URL, getPokemonChunk } from '@/pages/api/pokemon';
+import Typography from '../atoms/Typography';
+import Grid from '../layouts/Grid';
+import { useInfiniteQuery } from 'react-query';
+import { titleCase } from '@/utils/pokemon-names';
+import { useInView } from 'react-intersection-observer';
+import { useEffect } from 'react';
+import Link from 'next/link';
+
+export const mapPokemonDataToCardData = (
```

Comment:
I like this implementation, maybe you should put in in the utils

Reply:
The `CardDataProps` objects are tightly coupled to the `Card` component, so I created this separate mapping function in order to separate those details from the Pokémon API service methods. Didn't want to make the `getPokemonChunk()` function return data that is tightly coupled to a component's props because that would reduce the function's general reusability